fix(jans-fido2): compilation is failing due to missed jans-doc dependency

**Describe the bug**

![image](https://github.com/JanssenProject/jans/assets/2013070/e6d3e2bb-4446-4540-abcd-349cde1fdda0)

```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] Annotation processor 'io.jans.doc.annotation.DocPropertyProcessor' not found
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Fido2 API 1.0.14-SNAPSHOT:
[INFO] 
[INFO] Fido2 API .......................................... SUCCESS [  6.163 s]
[INFO] Fido2 API Model .................................... SUCCESS [ 12.980 s]
[INFO] Fido2 Client ....................................... SUCCESS [ 14.743 s]
[INFO] Jans notify client ................................. FAILURE [  0.033 s]
[INFO] Fido2 API Server ................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  34.299 s
[INFO] Finished at: 2023-05-22T08:25:09Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project jans-notify-client: Compilation failure
[ERROR] Annotation processor 'io.jans.doc.annotation.DocPropertyProcessor' not found
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
```

--- a/jans-fido2/notify-client/pom.xml
+++ b/jans-fido2/notify-client/pom.xml
@@ -27,5 +27,9 @@
 	        <groupId>org.jboss.resteasy</groupId>
 	        <artifactId>resteasy-jackson2-provider</artifactId>
 	    </dependency>
+        <dependency>
+            <groupId>io.jans</groupId>
+            <artifactId>jans-doc</artifactId>
+        </dependency>
    	</dependencies>
 </project>

